python/api_client.py

```diff
@@ -86,7 +86,7 @@ def send_project_for_edit(self, project_uuid: str, profile_key: str, crop: bool
     def _upload_image(self, file_upload_details: Dict):
         upload_link = file_upload_details['upload_link']
         file_name = file_upload_details['file_name']
-        headers = {}
+        headers = {"Content-Type": ""}
```

Comment:
People are struggling with this.. we need to be more clear that this is the content type they need to provide because when we generate presgined url we also don't specify it so in order for it work - it must be the same.